wordpress-mobile/AztecEditor-iOS
# Improve display of self-closing Gutenberg (Gutenpack) elements

This PR improves the display of the Gutenberg self closing elements instead of displaying just Gutenpack it show the name of the block.

It also improved the possible HTML edition of the same showing the content of the block instead of the encoded data.

To test:
 - Open the sample Gutenberg content
 - Scroll to the end (you may need to rotate the device to be able to scroll to the end, bug to fix) 
 - Check if you see WP:Latest-Posts
 - Tap on it
 - See the content of the block.
